README.md

```diff
@@ -1120,6 +1121,27 @@ Options for the [OpenID Connect](http://developer.okta.com/docs/api/resources/oi
     }
     ```
 
+### Smart Card IdP
+
+Options for authentication with a Smart Card `X509` type IdP.
+
+- `certAuthUrl` - Required url endpoint that requests the browser for a certificate.
+
+- `text` - Optional string that can be set as the Smart Card IdP button text. By default, this value will be "Sign in with PIV / CAC card".
+
+- `className` - Optional class that can be added to the Smart Card IdP button.
+
+- `isCustomDomain` - Optional boolean that is passed to the certAuthUrl endpoint to indicate whether the request comes from a custom domain.
+
+    ```javascript
+    piv: {
+      text: 'Authenticate with a Smart Card',
+      className: 'custom-style',
+      certAuthUrl: 'https://${yourOktaDomain}.okta.com/login/cert',
```

Comment:
Shouldn't this be `https://${yourOktaDomain}.okta.com/api/internal/v1/authn/cert`? `https://${yourOktaDomain}.okta.com/login/cert` will lead to the JSP instead.

@nikolaystoilov-okta will it be a problem if we reference an internal API here?

Reply:
Yea, this was just an example.. I wasn't sure if we could reference an internal api here. Will wait for Nikolay's input

Reply:
+1 on leaving it as is. This is just an example and we shouldn't make any reference to internal APIs in our doc

Reply:
ok, but it's a confusing example since `/login/cert` won't ever work...

Reply:
its a good point... but we don't have a public endpoint that will work, correct? We can make it more obvious that this is just an example with something like `/your/cert/validation/endpoint`?

Reply:
That might be an OK middle ground.

I assume that the customer will either need to contact Okta or see how the default SIW works to figure out the real URL.